tests/tom-swifty-test/XmlReflectionTests/XmlToTLFMappingTests.cs

```diff
@@ -268,12 +304,9 @@ public void CanFindClassReturnsClass ()
 			Func<FunctionDeclaration, bool> funcFinder,
 			Func<TLFunction, bool> tlVerifier)
 		{
-			CustomSwiftCompiler compiler = Utils.CompileSwift (code, moduleName: "CanFind");
-
-			var errors = new ErrorHandling ();
-			ModuleInventory mi = ModuleInventory.FromFile (Path.Combine (compiler.DirectoryPath, "libCanFind.dylib"), errors);
-			ModuleDeclaration mod = compiler.ReflectToModules (new string [] { compiler.DirectoryPath },
-				new string [] { compiler.DirectoryPath }, null, "CanFind") [0];
+			var inventoryModule = ReflectToModules (code, "CanFind");
+			var mi = inventoryModule.Item1;
+			var mod = inventoryModule.Item2;
```

Comment:
and you can call it like
```csharp
(var mi, var mod) = ReflectToModules (code, "CanFind");
```

Reply:
I didn't think I had access to that yet, but it looks like I do.
ALL THE TUPLES!